templates/glue-jobs.yaml

```diff
@@ -0,0 +1,146 @@
+
+  AnswerParquetJob:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      TemplateURL: !Sub https://${SourceBucketName}.s3.amazonaws.com/${BranchOrTagName}/templates/glue-job.yaml
+      Parameters:
+        JobDescription: Export answers data in parquet format
+        GlueDatabaseName: !Ref GlueDatabaseName
+        GlueTableName: answers
+        JobName: AnswersParquet
+        JobRole: !Ref JobRole
+        S3OutputBucketName: !Ref S3OutputBucketName
+        S3ScriptLocation: !Sub s3://${SourceBucketName}/${BranchOrTagName}/glue/jobs/json_s3_to_parquet.py
+
+  InfoParquetJob:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      TemplateURL: !Sub https://${SourceBucketName}.s3.amazonaws.com/${BranchOrTagName}/templates/glue-job.yaml
+      Parameters:
+        JobDescription: Export info data in parquet format
+        GlueDatabaseName: !Ref GlueDatabaseName
+        GlueTableName: info
+        JobName: InfoParquet
+        JobRole: !Ref JobRole
+        S3OutputBucketName: !Ref S3OutputBucketName
+        S3ScriptLocation: !Sub s3://${SourceBucketName}/${BranchOrTagName}/glue/jobs/json_s3_to_parquet.py
+
+  MetadataParquetJob:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      TemplateURL: !Sub https://${SourceBucketName}.s3.amazonaws.com/${BranchOrTagName}/templates/glue-job.yaml
+      Parameters:
+        JobDescription: Export metadata data in parquet format
+        GlueDatabaseName: !Ref GlueDatabaseName
+        GlueTableName: metadata
+        JobName: MetadataParquet
+        JobRole: !Ref JobRole
+        S3OutputBucketName: !Ref S3OutputBucketName
+        S3ScriptLocation: !Sub s3://${SourceBucketName}/${BranchOrTagName}/glue/jobs/json_s3_to_parquet.py
+
+  MicrophoneLevelsParquetJob:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      TemplateURL: !Sub https://${SourceBucketName}.s3.amazonaws.com/${BranchOrTagName}/templates/glue-job.yaml
+      Parameters:
+        JobDescription: Export microphone levels data in parquet format
+        GlueDatabaseName: !Ref GlueDatabaseName
+        GlueTableName: microphone_levels
+        JobName: MicrophoneLevelsParquet
+        JobRole: !Ref JobRole
+        S3OutputBucketName: !Ref S3OutputBucketName
+        S3ScriptLocation: !Sub s3://${SourceBucketName}/${BranchOrTagName}/glue/jobs/json_s3_to_parquet.py
+
+  MotionParquetJob:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      TemplateURL: !Sub https://${SourceBucketName}.s3.amazonaws.com/${BranchOrTagName}/templates/glue-job.yaml
+      Parameters:
+        JobDescription: Export motion data in parquet format
+        GlueDatabaseName: !Ref GlueDatabaseName
+        GlueTableName: motion
+        JobName: MotionParquet
+        JobRole: !Ref JobRole
+        S3OutputBucketName: !Ref S3OutputBucketName
+        S3ScriptLocation: !Sub s3://${SourceBucketName}/${BranchOrTagName}/glue/jobs/json_s3_to_parquet.py
```

Comment:
So each of these jobs will create the different glue tables?

Reply:
No, the job names indicate which Glue table each job reads as input. They then relationalize the schema as defined in the glue table, producing multiple, flat tables, and writing each table to its own parquet dataset.

Reply:
also, a tables PR will be forthcoming next